js/phobos.js

```diff
@@ -23,14 +23,28 @@ function addAnchors()
         // we link to the first children
         var a = items[i].querySelector('a');
         if(!a) continue;
+
+        // copy-to-clipboard
+        var el = document.createElement("i");
+        var symbol = a.name.replace(".", "");
```

Comment:
BTW, doesn't work for methods, e.g. `import std.math : FloatingPointControl.severeExceptions;`.